test/infoSpec.js

```diff
@@ -129,6 +130,133 @@ describe('Info', () => {
     })
   })
 
+  describe('authentication', () => {
+    beforeEach(async function () {
+      this.newSocket = new MockSocket()
+      await this.plugin.addSocket(this.newSocket)
+    })
+
```

Comment:
Check that all `MockSocket` handlers have been called by adding `afterEach`:

```js
  afterEach(async function () {
    assert(await this.mockSocket.isDone(), 'request handlers must have been called')
  })
```

Reply:
added